shared/components/modals/WithdrawModal/WithdrawModal.js

```diff
@@ -153,13 +173,14 @@ export default class WithdrawModal extends React.Component {
       })
   }
 
-    sellAllBalance = () => {
+    sellAllBalance = async () => {
       const { amount, balance, currency, tokenFee } = this.state
       const { data } = this.props
-      const minFee = tokenFee ? Number(0) : minAmount[data.currency.toLowerCase()]
+
+      const minFee = tokenFee ? 0 : minAmount[data.currency.toLowerCase()]
 
       const balanceMiner = balance !== 0
-        ? Number(balance) - minFee
+        ? new BigNumber(balance).minus(minFee).toNumber()
```

Comment:
Это важный момент, так как при переводе в Number число неизбежно округляется, а в String нет.

```suggestion
        ? new BigNumber(balance).minus(minFee).toString()
```